mapic/stats_collector.go

```diff
@@ -77,6 +77,9 @@ func (c *metricsCollector) collectMetrics(ctx context.Context) error {
 		if err := ctx.Err(); err != nil {
 			return err
 		}
+		if streamID == "" {
```

Comment:
When might this happen?

Reply:
Good question. TBH Not sure right now. I think I added it because (for some reason) Mist returned `""` stream ID which caused panic.